GOVERNANCE.md

```diff
@@ -201,7 +201,14 @@ Current Core Contributors can nominate candidates for consideration by the counc
 can make the determination for acceptance with a process of their choosing.
 
 #### Current Core Contributors
-* Will be updated with Core Contributor list during first election
+* Alex Andorra (@AlexAndorra)
+* Colin Carroll (@ColCarroll)
+* Ari Hartikainen (@ahartikainen)
+* Ravin Kumar (@canyon289)
+* Osvaldo Martin (@aloctavodia)
+* Du Phan (@fehiepsi)
+* Oriol Abril Pla (@OriolAbril) 
```

Comment:
I did not notice this was alphabetical order at first, we have two last names and I am still young enough to try and keep them both :see_no_evil:. It will probably be better to reduce confusion to use `Oriol Abril-Pla`